pe-utils/deb/debian/pelion.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Pelion Edge Multi-Service Systemd Control
+After=network-online.target
+
+[Service]
+Type=oneshot
+ExecStart=/bin/true
+RemainAfterExit=yes
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Does it makes sense to make this a [target](https://www.freedesktop.org/software/systemd/man/systemd.target.html) instead of a service since the `[Service]` stuff is just a no-op(`/bin/true`)?

Reply:
Hrmmm... I'm not sure you want a target here though since targets typically represent the old sysV runlevels